src/views/tables/OrdersTable.tsx

```diff
@@ -147,11 +147,11 @@ const getOrdersTableColumnDef = ({
           </TableColumnHeader>
         ),
         tag: symbol,
-        renderCell: ({ size, totalFilled, stepSizeDecimals }) => (
+        renderCell: ({ remainingSize, size, totalFilled, stepSizeDecimals }) => (
           <TableCell stacked>
             <Output
               type={OutputType.Asset}
-              value={size}
+              value={remainingSize ?? size}
```

Comment:
ah thought about it, decided to revert this change and update the chart lines to show size instead of remaining size [here](https://github.com/dydxprotocol/v4-web/pull/778/commits/4bcb6eeb848026c43bfd68cc4edf233ef147ef66#diff-b161cc90dfb04db4caa08ad0ef67f20e3be68848f1e4f04b1c14bd8879ea6abaL205) for consistency